src/Message/CommandHandler/CreateNumberMessageHandler.php

```diff
@@ -5,17 +5,28 @@
 namespace App\Message\CommandHandler;
 
 use App\Message\Command\CreateNumber;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Messenger\Handler\MessageSubscriberInterface;
 
 /**
  * This command handler handles message asynchronously. See config in config/packages/framework.yaml
  * A CommandHandler (Command as in CQRS) should never return anything.
  *
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-class CreateNumberHandler
+class CreateNumberMessageHandler implements MessageSubscriberInterface
 {
-    public function __invoke(CreateNumber $command)
+    /**
+     * {@inheritdoc}
+     */
+    public static function getHandledMessages(): array
     {
+        return [CreateNumber::class];
+    }
+
+    public function __invoke($command)
```

Comment:
We remove the type hint?

Reply:
It's still possible to use it (it is in the query part). But it's to show it uses the interface :)